### PR TITLE
[Feature] Polish Unstoppable Domain error curation

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Goal
Improve the user experience by giving a meaningful response when the user gets no domain search

## Approach
K.I.S.S. Users rarely have time or desire to relay error codes. They just want it to work. So the codes are reduced from search errors and everything else is passed in the analytics

## Steps
- Localize the errors
- Add model to map the UD errors to a subset of easy to read messages
- Add simple codes
- Change the toast color that is noticeable yet not alarming

## Video Clips

Before | After
-------|------
before-uderrro|after-uderrors

### CODE REVIEW

**Comment by kcw-grunt** [comment]
Holding until next week

**Comment by kcw-grunt** [comment]
Ready for review

File: loafwallet/DomainResolutionFailure.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  
+    init() {}
+    
+    func messageWith(error: ResolutionError) -> String {
+        
+        switch error {
+            case .unregisteredDomain,.unsupportedDomain,
+                 .recordNotFound,.recordNotSupported,
+                 .unspecifiedResolver:
+                return String(format:  S.Send.UnstoppableDomains.lookupDomainError, 0)
```
Comment (mosadialiou) [inline loafwallet/DomainResolutionFailure.swift:22]: @kcw-grunt Just wondering. What returns for english device 
`String(format:  S.Send.UnstoppableDomains.lookupDomainError, 0)`?  Is it `Sorry, domain was not found. [Error: 0]`?
If yes what's the meaning of `[Error: 0]` and `[Error: 10]` ?

Comment (kcw-grunt) [reply loafwallet/DomainResolutionFailure.swift:22]: This is the sorting the group of errors to make it easy for customer support....we track events...but right now the load is 21 requests per week.....so all of this if very low priority.

Comment (mosadialiou) [reply loafwallet/DomainResolutionFailure.swift:22]: OK noted.
As I told you yesterday, I will do my best to review all the PRs. So this comment was just for that.

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**